nohup: ignoring input
During startup - Warning messages:
1: Setting LC_CTYPE failed, using "C" 
2: Setting LC_COLLATE failed, using "C" 
3: Setting LC_TIME failed, using "C" 
4: Setting LC_MESSAGES failed, using "C" 
5: Setting LC_MONETARY failed, using "C" 
6: Setting LC_PAPER failed, using "C" 
7: Setting LC_MEASUREMENT failed, using "C" 
-- Attaching core tidyverse packages ------------------------ tidyverse 2.0.0 --
v dplyr     1.1.4     v readr     2.1.5
v forcats   1.0.0     v stringr   1.5.1
v ggplot2   3.5.2     v tibble    3.2.1
v lubridate 1.9.4     v tidyr     1.3.1
v purrr     1.0.4     
-- Conflicts ------------------------------------------ tidyverse_conflicts() --
x dplyr::between()     masks data.table::between()
x dplyr::filter()      masks stats::filter()
x dplyr::first()       masks data.table::first()
x lubridate::hour()    masks data.table::hour()
x lubridate::isoweek() masks data.table::isoweek()
x dplyr::lag()         masks stats::lag()
x dplyr::last()        masks data.table::last()
x lubridate::mday()    masks data.table::mday()
x lubridate::minute()  masks data.table::minute()
x lubridate::month()   masks data.table::month()
x lubridate::quarter() masks data.table::quarter()
x lubridate::second()  masks data.table::second()
x purrr::transpose()   masks data.table::transpose()
x lubridate::wday()    masks data.table::wday()
x lubridate::week()    masks data.table::week()
x lubridate::yday()    masks data.table::yday()
x lubridate::year()    masks data.table::year()
i Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors
TwoSampleMR version 0.6.14 

  [>] New authentication requirements: https://mrcieu.github.io/ieugwasr/articles/guide.html#authentication.
  [>] Major upgrades to our servers completed to improve service and stability.
  [>] We need your help to shape our emerging roadmap!
      Please take 2 minutes to give us feedback -
      https://forms.office.com/e/eSr7EFAfCG
Warning message:
package 'TwoSampleMR' was built under R version 4.5.0 
Error in dyn.load(file, DLLpath = DLLpath, ...) : 
  unable to load shared object '/mnt/sas0/AD/rmgupte/R/x86_64-pc-linux-gnu-library/4.2/plyr/libs/plyr.so':
  /usr/local/lib64/libstdc++.so.6: version `GLIBCXX_3.4.32' not found (required by /mnt/sas0/AD/rmgupte/R/x86_64-pc-linux-gnu-library/4.2/plyr/libs/plyr.so)
Calls: %>% ... format_data -> loadNamespace -> library.dynam -> dyn.load
Execution halted
